File: python/rna-transcription/rna_transcription.py
def to_rna(dna_strand):
    rna = ""
    for dna in dna_strand.upper():
        if dna == 'G':
            rna += 'C'
        if dna == 'C':
            rna += 'G'
        if dna == 'T':
            rna += 'A'
        if dna == 'A':
            rna += 'U'
    return rna

File: go/rna-transcription/rna_transcription.go
package strand

import "strings"

func ToRNA(dna string) string {
	rna := strings.NewReplacer(
		"G", "C",
		"C", "G",
		"T", "A",
		"A", "U",
	)
	return rna.Replace(dna)
}
